### DISCUSSION TITLE
Inline styles in templates creating media queries when you add the block to the editor

### DISCUSSION THREAD

**Discussion by jcenlo** [GrapesJS/grapesjs#5008] [Q&A] [answered]
I'm building a theme with many blocks, this is an example of a block.
``` jsx
import { renderToString } from "react-dom/server";

export const About_04 = (
  <section className="wrapper py-15">
    <div className="container">
      <div className="row d-flex align-items-center">
        <div
          className="figure col-7 position-absolute z-index_0 "
          // here is the problem.............
          style={{ right: 0, top: 0 }}
          //...........................................
        >
          <img
            alt="Image"
            className="img-fluid rounded overflow-hidden shadow"
            src="https://dummyimage.com/1000x1000/999/222"
          />
        </div>
      </div>
    </div>
  </section>
);

export const About_04_block = (bm) => {
  bm.add("about_04", {
    label: "About 04",
    category: "About Aby",
    content: renderToString(About_04),
  });
};

```
The example contains divs with inline styles like this one.
<img width="526" alt="Screenshot 2023-03-24 at 16 51 59" src="https://user-images.githubusercontent.com/1485745/227575789-297884bb-0c77-482e-98fa-31f12b4555f8.png">

When using it on the canvas, I realized that it transformed the inline styles into an ID and added the styles to an external class. The problem is that, these generated styles are generated with a media query belonging to the one you have selected.

<img width="1039" alt="Screenshot 2023-03-24 at 16 52 55" src="https://user-images.githubusercontent.com/1485745/227576052-3179c74b-93c8-4124-8496-a8bff8b3d01a.png">
<img width="663" alt="Screenshot 2023-03-24 at 16 53 58" src="https://user-images.githubusercontent.com/1485745/227576782-7321b855-ce31-42b4-8f53-032fe1264314.png">

Is there any way or method to apply the styles to all media queries? or avoid config of media query, i understand why this could work like that but, Is there any method to solve this?

**Comment by jcenlo** [comment, marked as answer]
@artf i am not sure if this is a bug or a bad UX, because not a clear or expected action, i can understand why can works like this, but i am not sure if this method is the best solution.

I am UX / UI / DEV i am happy to help if you need it, this project is wonderful 

also i am trying this in the grapescli and i am not experimenting the same issue, maybe is my code

**Reply by artf** [reply]
Yeah right now inline styles are always created with the current selected device, what I'd suggest is moving those styles in a class name

**Reply by jcenlo** [reply]
Ok i just discover what was my problem, i re-wrote the [Default  Desktop Device](https://github.com/GrapesJS/grapesjs/blob/master/src/device_manager/config/config.ts) so always was adding a media-query css, and bigger or smaller Devices was restricted of that styles
<img width="335" alt="Screenshot 2023-04-03 at 14 30 47" src="https://user-images.githubusercontent.com/1485745/229510235-3c79d59a-8bbd-4efa-a10b-2886b31af670.png">

I think this could be notice in the docs, i have to learn how to i can collaborate in this kind of things.

Thanks any way.
